site/app/views/submission/HomeworkView.php

```diff
@@ -178,16 +178,15 @@ public function showGradeable($gradeable, $days_late) {
                 $image_width = $image_height = 0;
 
                 if (isset($gradeable->getTextboxes()[$i]['images']) && $gradeable->getTextboxes()[$i]['images'] != ""){
-                    $tester = $gradeable->getTextboxes()[$i]['images'];
+                    $images = $gradeable->getTextboxes()[$i]['images'];
                 }
                 else{
-                    $tester = array();
+                    $images = array();
                 }
 
-                //
-                foreach((array)$tester as $currImage){
+                foreach((array)$images as $currImage){
```

Comment:
Same comment as I made in #1250, there's no reason to have this cast to `(array)` here.